src/components/Layout/Layout.stories.tsx

```diff
@@ -152,6 +169,52 @@ function TableExample({ numCols = 10, numRows = 100 }: { numCols?: number; numRo
   );
 }
 
+function VirutalizedPage() {
+  const { pl } = useScrollableParent();
+  return (
+    <>
+      <TestHeader title="Change Event - Mud Room" />
+      <p>Content above the table</p>
+      {/*
+        Providing a <ScrollableContent> wrapper here isn't really necessary,
+        as the Virtualized component will provides its own scrolling behavior.
```

Comment:
I guess I thought, even if the virtual table does its own scrolling, we'd still want ScrollableContent to shove the table up "into the right spot" inside ScrollableParent, but I guess ScrollableParent is setup a bit differently, where the content-to-be-scrolled doesn't have to be in a specific div, it can just come after the fixed content...